### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  com.microsoft.azure.toolkit.lib.auth.AzureToolkitAuthenticationException: you are not signed-in.

### ISSUE THREAD

**Issue by dinbtechit** [microsoft/azure-tools-for-java#6930]
IntelliJ build version: 2022.2.1 IU-222.3739.54
OS: Mac OS X
JDK: JetBrains s.r.o. 17.0.3
Plugin version: 3.67.0-2022.2
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,0,1920x1010,invalid,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=com.intellij.ui.ToolbarUtil$3@34cafce5,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=com.microsoft.azure.toolkit.lib.auth.AzureToolkitAuthenticationException: you are not signed-in.
	at com.microsoft.azure.toolkit.lib.auth.AzureAccount.lambda$account$0(AzureAccount.java:54)
	at com.microsoft.azure.toolkit.lib.auth.AzureAccount.account(AzureAccount.java:54)
	at com.microsoft.azure.toolkit.intellij.legacy.appservice.AppServiceInfoBasicPanel.init(AppServiceInfoBasicPanel.java:64)
	at com.microsoft.azure.toolkit.intellij.legacy.appservice.AppServiceInfoBasicPanel.<init>(AppServiceInfoBasicPanel.java:60)
	at com.microsoft.azure.toolkit.intellij.legacy.function.FunctionAppCreationDialog$1.<init>(FunctionAppCreationDialog.java:58)
	at com.microsoft.azure.toolkit.intellij.legacy.function.FunctionAppCreationDialog.createUIComponents(FunctionAppCreationDialog.java:58)
	at com.microsoft.azure.toolkit.intellij.legacy.functi

**Comment by wangmingliang-ms** [comment]
thanks for reporting, this bug should be fixed by commit  cf338951, and the fix would be included in the next stable release.

**Comment by wangmingliang-ms** [comment]
close as it has been fixed since v3.69.0, you can upgrade to the latest and try. Feel free to reopen it when you encounter this error again.
